pkg/services/store/entity/dummy/dummy_server.go

```diff
@@ -1,398 +0,0 @@
-package dummy
-
-import (
-	"context"
-	"crypto/md5"
-	"encoding/hex"
-	"encoding/json"
-	"errors"
-	"fmt"
-	"strconv"
-	"time"
-
-	"github.com/grafana/grafana/pkg/infra/log"
-	"github.com/grafana/grafana/pkg/infra/x/persistentcollection"
-	"github.com/grafana/grafana/pkg/services/grpcserver"
-	"github.com/grafana/grafana/pkg/services/store"
-	"github.com/grafana/grafana/pkg/services/store/entity"
-	"github.com/grafana/grafana/pkg/services/store/kind"
-	"github.com/grafana/grafana/pkg/setting"
-)
-
-type EntityVersionWithBody struct {
-	*entity.EntityVersionInfo `json:"info,omitempty"`
-
-	Body []byte `json:"body,omitempty"`
-}
-
-type EntityWithHistory struct {
-	Entity  *entity.Entity           `json:"entity,omitempty"`
-	History []*EntityVersionWithBody `json:"history,omitempty"`
-}
-
-var (
-	// increment when Entity changes
-	rawEntityVersion = 10
-)
-
-// Make sure we implement both store + admin
-var _ entity.EntityStoreServer = &dummyEntityServer{}
-var _ entity.EntityStoreAdminServer = &dummyEntityServer{}
-
-func ProvideDummyEntityServer(cfg *setting.Cfg, grpcServerProvider grpcserver.Provider, kinds kind.KindRegistry) entity.EntityStoreServer {
```

Comment:
This isn't needed anymore now that the SQL one works for integration tests -- keeping it in sync will will be pretty hard as we evolve the response shapes (and add things like watch!)